admin/includes/admin_html_head.php

```diff
@@ -22,12 +22,12 @@
     <link rel="stylesheet" href="<?php echo $value; ?>">
     <link rel="stylesheet" href="includes/fontawesome/css/solid.min.css">
     <link rel="stylesheet" href="includes/fontawesome/css/regular.min.css">
-    <?php if (defined('FONTAWESOME_V4_SHIM') && FONTAWESOME_V4_SHIM == 'true' && file_exists($value = 'includes/fontawesome/css/v4-shims.min.css')) { ?>
+    <?php if (isset($fontawesome_v4_shim_required) && $fontawesome_v4_shim_required == true && file_exists($value = 'includes/fontawesome/css/v4-shims.min.css')) { ?>
```

Comment:
Use `!empty($fontawesome_v4_shim_required)` instead of `isset` and the follow-on test.

Reply:
I checked https://www.php.net/manual/en/function.defined.php when making the change, which states "If you want to see if a variable exists, use isset() as defined() only applies to constants" .. `empty`does not seem a good fit to me, see https://www.php.net/manual/en/function.empty.php

Reply:
PR updated.